builtins/en/steps/reviewers.yaml

```
@@ -1,110 +1,73 @@
 name: reviewers
-tags: [review]
+tags: [ review ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)